src/components/Tournament/index.tsx

```diff
@@ -54,13 +61,14 @@ const Tournament = () => {
 
   const handleConfirmClick = () => {
     setLoading(true);
-    firebase.saveTournament(tournament).then(() => {
-      setTournament({ schedule: [], name: "" });
-      setLoading(false);
-      setTournamentName("");
-      setSelectedTeams([]);
-      //TODO redirect user to tournament/tournamentID page to see scores
-    });
+    tournament &&
+      firebase.saveTournament(tournament).then(() => {
+        setTournament(null);
+        setLoading(false);
+        setTournamentName("");
+        setSelectedTeams([]);
+        //TODO redirect user to tournament/tournamentID page to see scores
+      });
   };
 
   return (
```

Comment:
I think the outmost fragment in this file is not needed.

Reply:
What fragment?